packages/server/src/index.ts

```diff
@@ -103,12 +104,16 @@ class Server implements ServerType {
         app = express()
 
         app.use((req, res, next) => {
-          req.pipe(
-            concat((data: any) => {
-              req.body = data.toString()
-              next()
-            })
-          )
+          // Convert the Reqest (extends <Stream.Readable>) into a
+          // WHATWG Web Standard ReadableStream
+          req.body = new WebReadableStream({
+            start: (controller) => {
+              req.on('data', controller.enqueue)
+              req.on('close', controller.close)
+              req.on('error', controller.error)
+            },
+          })
+          next()
```

Comment:
Another solution that seemed to work (surprisingly) was just to go:

```
req.body = req
next()
```

However I thought this approach was more appropriate since it maintains Web standard streams.